pyproject.toml

```diff
@@ -48,6 +48,18 @@ SQLAlchemy = "<2.0"
 tables = "^3.7.0"
 watchdog = "^3.0.0"
 containers-sugar = "1.9.1"
+django-celery-beat = "^2.5.0"
+redis = "^5.0.1"
+django-redis = "^5.4.0"
+episcanner-downloader = { git = "https://github.com/luabida/episcanner-downloader", branch = "update-deps" }
```

Comment:
I think this will not be a necessary package to install as a main system dependency, maybe it would be better to separate it in a new group within pyproject.